sale_stock_medical_prescription/demo/medical_medicament_demo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="product_product_amoxicillin_1" model="product.product">
+        <field name="name">Amoxicillin</field>
+        <field name="uom_id" ref="product.product_uom_unit" />
+        <field name="uom_po_id" ref="product.product_uom_unit" />
+        <field name="weight">0.00025</field>
+        <field name="categ_id" ref="sale_medical_prescription.product_category_rx" />
+        <field name="description_sale">Penicillin Capsule 250mg</field>
+        <field name="pricelist_id" ref="product.list0" />
+        <field name="standard_price">0.85</field>
+        <field name="list_price">1.50</field>
+        <field name="currency_id" ref="base.USD" />
+        <field name="default_code">OMCLN</field>
+        <!-- Commented out for hooks testing
```

Comment:
Yes it's commented out to show it should have a value of 'product' (which the post_init_hook will do). I should probably adjust that thought it's quite unclear why I've commented it out : |, looks at first glance like a WIP comment.